routers/api/v1/user/repo.go

```diff
@@ -66,6 +70,20 @@ func ListMyRepos(ctx *context.APIContext) {
 	listUserRepos(ctx, ctx.User)
 }
 
+// ListOrgRepos - list the repositories of an organization.
+func ListOrgRepos(ctx *context.APIContext) {
+	// swagger:route GET /orgs/{org}/repos orgListRepos
+	//
+	//     Produces:
+	//     - application/json
+	//
+	//     Responses:
+	//       200: RepositoryList
+	//       500: error
+
+	listUserRepos(ctx, ctx.Org.Organization)
```

Comment:
We may need to check usertype of ctx.Org.Organization.Type the same could maybe done under /users/{username}/repo. Maybe tests could be added to check those corner cases.